nerfstudio/models/depth_nerfacto.py

```diff
@@ -118,7 +120,7 @@ def get_image_metrics_and_images(
             far_plane=torch.max(ground_truth_depth),
         )
         images["depth"] = torch.cat([ground_truth_depth_colormap, predicted_depth_colormap], dim=1)
-        metrics["depth_mse"] = torch.nn.functional.mse_loss(outputs["depth"], ground_truth_depth)
+        metrics["depth_mse"] = torch.nn.functional.mse_loss(outputs["depth"]*(ground_truth_depth>0), ground_truth_depth)
```

Comment:
You need to mask the ground truth as well. Otherwise the zeros will always be correct and your metric will be biased high.

Reply:
```mse_loss(outputs["depth"][depth_mask], ground_truth_depth[depth_mask])```